### PR TITLE
update sop for refresh imports

### LINKED ISSUE #6988: Update Mondo docs on refreshing imports
Update the Mondo docs to state that that command to run to refresh imports is: `sh run.sh make refresh-merged`. This should replace the information "Instructions" https://mondo.readthedocs.io/en/latest/editors-guide/imports/#instructions

### PR DESCRIPTION
**Pull request by twhetzel**
Closes #6988 

Update the docs to refresh imports since Nico mentioned earlier this is outdated.

### CODE REVIEW

**Review by matentzn — APPROVED** [review]
Great!

File: docs/editors-guide/imports.md
Comment (sabrinatoro) [inline docs/editors-guide/imports.md:24]: typo: "newly deprecatecd classes" should be "newly deprecated classes"

**Review by sabrinatoro — APPROVED** [review]
@twhetzel 2 small things:
1) there is a typo (see comment)
2) I am "assuming" the instructions work, but I will know if something is missing (especially for people who might need extra explanations) when I try to refresh the imports on my own.
3) Most (if not all) curators use GitHub Desktop. It would be helpful to have instructions using GitHub Desktop. 

All that being said, I think point 2 and 3 could be taken care of at a later point, and this PR can be merged after the typo has been fixed.

File: docs/editors-guide/imports.md
Comment (twhetzel) [reply docs/editors-guide/imports.md]: thanks for the comment on the typo. i did not re-write the entire step and missed that.

Comment (twhetzel) [reply docs/editors-guide/imports.md]: these are the instructions from Nico

Comment (twhetzel) [reply docs/editors-guide/imports.md]: The git steps are the original ones for this page. As part of the Documentation clean-up and re-organization, I'd suggest there be a page with both GitHub Desktop and git command line that briefly goes over a few common actions (https://github.com/monarch-initiative/mondo/issues/7015) and be referenced in the pages as needed for common git commands.

**Comment by twhetzel** [comment]
As Reviewer, are you going to merge or do you want me to add myself as a Reviewer and Approve and Merge vs. using the bypass?

File: docs/editors-guide/imports.md
Comment (twhetzel) [reply docs/editors-guide/imports.md]: Also, for the future it's fine to hold off on Approval until you have time to run the command and verify.

**Comment by sabrinatoro** [comment]
@twhetzel you should be able to merge.

**Comment by twhetzel** [comment]
Thanks for getting the PR back into a state it can be merged without bypassing checks following the update for the typo. I will merge this, but I thought Reviewers were merging things other than code updates.

**Comment by twhetzel** [comment]
Verified the page is updated: https://mondo.readthedocs.io/en/latest/editors-guide/imports/#regenerate-import-file

**Pull request merged**
